_data/navigation.yml

```diff
@@ -16,7 +16,9 @@
 - text: AirCasting
   url: /aircasting
   subitems:
-    - text: Aircasting App
+    - text: AirCasting Maps
+      url: http://aircasting.habitatmap.org/mobile_map
+    - text: AirCasting App
       url: https://play.google.com/store/apps/details?id=pl.llp.aircasting&hl=en_US
```

Comment:
Do we have a way of making this open in a new tab?

Reply:
links are generated in a loop, it's not complicated to add an extra field and conditionally check if the link should be external and add the attribute if so, but it needs setting up.